run-make-check.sh

```diff
@@ -76,29 +84,37 @@ function run() {
 
     CMAKE_BUILD_OPTS="-DWITH_FIO=ON -DWITH_GTEST_PARALLEL=ON"
 
-    # Are we in the CI ?
+    cat <<EOM
+Note that this script modifies $HOME/.ccache/ccache.conf to ensure maximum
+performance. If that is not desirable, consider running the script as a
+different user. Also note that the binaries produced by this script do not
+contain correct time and git version information, which may make them
+unsuitable for debugging and production use.
+EOM
+    # remove the entropy generated by the date/time embedded in the build
+    CMAKE_BUILD_OPTS="$CMAKE_BUILD_OPTS -D ENABLE_GIT_VERSION=OFF"
+    export SOURCE_DATE_EPOCH=$(date +%D |date -f- +%s)
```

Comment:
Please stop using %D. It's not as portable as you think. coreutils does hardcode it, but it's been locale dependent before. Additionally the variant I offered is more readible: `$(TZ=UTC date -d 'today 00:00' +%s)`

Reply:
@robbat2 Your point have been taken in account and @LenzGr was supporting that vision too. The pending question is more "what should be the default epoch for best performance". I'm wondering if 0 could be the answer meaning that we don't need the date thing.

Reply:
Setting `SOURCE_DATE_EPOCH` to zero would be fine with me. In the end, this is for speeding up test builds, not for generating production/release builds.